src/Stripe.net/Services/Files/FileService.cs

```diff
@@ -24,22 +24,26 @@ public FileService(string apiKey)
 
         public virtual File Create(FileCreateOptions options, RequestOptions requestOptions = null)
         {
+            requestOptions = this.SetupRequestOptions(requestOptions);
+            requestOptions.BaseUrl = StripeConfiguration.FilesBase;
             return Mapper<File>.MapFromJson(
                 Requestor.PostFile(
-                    this.ClassUrl(StripeConfiguration.FilesBase),
+                    StripeConfiguration.FilesBase + this.ClassUrl(),
```

Comment:
Technically, setting `requestOptions.BaseUrl` is useless here because the method doesn't call `Service.Request`. That's only temporary because `Service.Request` doesn't support multipart encoding yet, but I'll fix that later. In the meantime, this ensure that `requestOptions.BaseUrl` is consistent with the URL that is passed to `Requestor.PostFile`.